### ISSUE TITLE
could you please publish the schematic for this board?

### ISSUE THREAD

**Issue by jasonblewis** [LilyGO/TTGO-T-Eight-ESP32#2]
It would be great if we had a schematic for this board

thanks

**Comment by hak8or** [comment]
@LilyGO  I agree, can we please get a schematic? If not, then at least what the closest board to this is which also has a schematic released? For example, I would like to add support to esphome for this board, but I have no clue what board to use or even start from.

Closest I can get is the wrover kit? https://docs.platformio.org/en/latest/boards/espressif32/esp-wrover-kit.html
